src/Tasks/Microsoft.ManagedLanguages.targets

```diff
@@ -0,0 +1,64 @@
+﻿<!--
+***********************************************************************************************
+WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and have
+          created a backup copy.  Incorrect changes to this file will make it
+          impossible to load or build your projects from the command-line or the IDE.
+
+This file defines common build logic for all managed languaged: C#, VisualBasic, F#
```

Comment:
@nguerrera @livarcocc and maybe @jaredpar or @agocke: any objections to introducing this new common targets file, intended to be imported from `Microsoft.CSharp.targets` and similar for other languages?

Reply:
No real concerns, but let me bikeshed on the name while I'm here. :)

There are two analogs to this already:

* [Microsoft.Managed.Core.targets](https://github.com/dotnet/roslyn/blob/master/src/Compilers/Core/MSBuildTask/Microsoft.Managed.Core.targets)
* [Microsoft.Managed.DesignTime.targets](https://github.com/dotnet/project-system/blob/master/src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/DesignTimeTargets/Microsoft.Managed.DesignTime.targets)

Following an emergent pattern, this one should be Microsoft.Managed.targets.

The pattern I see is X.targets -> msbuild, X.DesignTime.targets -> project-system., X.Core.targets -> roslyn

Reply:
Good observation. I'll rename it to `Microsoft.Managed.targets`